support/kwok/kwok.go

```diff
@@ -89,8 +100,11 @@ func (k *Cluster) getKubeconfig() (string, error) {
 	kubecfg := fmt.Sprintf("%s-kubecfg", k.name)
 
 	var stdout, stderr bytes.Buffer
-	err := utils.RunCommandWithSeperatedOutput(fmt.Sprintf(`%s get kubeconfig --name %s`,
-		k.path, k.name), &stdout, &stderr)
+	cmd := fmt.Sprintf(`%s get kubeconfig --name %s`, k.path, k.name)
+	if len(k.getKubeconfigArgs) > 0 {
+		cmd = fmt.Sprintf("%s %s", cmd, strings.Join(k.getKubeconfigArgs, " "))
+	}
```

Comment:
strings.Join() already has validation your provided and seems to it's not needed here
```
// Join concatenates the elements of its first argument to create a single string. The separator
// string sep is placed between elements in the resulting string.
func Join(elems []string, sep string) string {
	switch len(elems) {
	case 0:
		return ""
```

```suggestion
	cmd := fmt.Sprintf(`%s get kubeconfig %s --name %s`, k.path, strings.Join(k.getKubeconfigArgs, " "), k.name)
```

Reply:
Thanks. Lets tests githubs and prow's ability to deal with your propose change :)

Reply:
Oh, current tests are flaky, I've never seen to pass all tests.